var/spack/repos/builtin/packages/elpa/package.py

```diff
@@ -29,7 +29,7 @@ class Elpa(AutotoolsPackage, CudaPackage):
     version('2015.11.001', sha256='c0761a92a31c08a4009c9688c85fc3fc8fde9b6ce05e514c3e1587cf045e9eba')
 
     variant('openmp', default=False, description='Activates OpenMP support')
-    variant('optflags', default=True, description='Build with optimization flags')
+    variant('optflags', default=True, description='Build with -O2 optimization flags. GCC only.')
```

Comment:
Is there a reason for activating `-O2` only for GCC? Even though the optimizations may be different across compilers and versions of the same compiler, `-O2` is supposed to be a generic flag understood by all compilers.

Reply:
Ok, thanks for the explanation - I fully agree. What was not clear to me was the:

> -O2 and -ffree-line-length-none are GCC specific.

statement in the description. 

Since there's no maintainer to ask, and we both agree that the closer we stay to upstream developers build system the better it is, can you drop the `optflags` variant completely (and related options) and see if we can build? I can also help testing with a few GCC versions. The option was added back in #5306 and there are also a few links to have a look at in a comment there. Thanks!